### PR TITLE
 chore : added C++ and DSA course

### LINKED ISSUE #1893: [Add] < Want to add Youtube link for C++ and data structures tutorial>
### Site URL

https://www.youtube.com/watch?v=WQoB2z67hvY&list=PLDzeHZWIZsTryvtXdMr6rPh4IDexB5NIA

### Description of the above provided site

This Youtube tutorial offers a beginner friendly guide to learn C++ and covers all the Data structures and Algorithms (DSA) required for placements.

### Category

Languages

### Sub Category

C++ and DSA

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by Romit77**
## Fixes Issue
 Closes #1893 

## Changes proposed

added a new link under languages category

## Screenshots

image

## Note to reviewers

This would be a great addition to the resources

### CODE REVIEW

File: database/languages/cpp.json [hunk 1]
```diff
@@ -12,5 +12,12 @@
     "url": "https://www.youtube.com/watch?v=8jLOx1hD3_o",
     "category": "languages",
     "subcategory": "cpp"
+  },
+  {
+    "name": "Complete C++ and Data Structures and Algorithms tutorial",
+    "description": "This Youtube tutorial offers a beginner friendly guide to learn C++ all the Data structures and Algorithms required for placements.",
```
Comment (Anmol-Baranwal) [inline database/languages/cpp.json:18]:
```suggestion
    "description": "This YouTube tutorial offers a beginner-friendly guide to learning C++, including all the data structures and algorithms required for placements.",
```

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
Just minor changes in the description

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by CBID2 — APPROVED** [review]
Looks great @Romit77

**Pull request merged**